packages/babel-plugin-minify-constant-folding/src/index.js

```diff
@@ -112,6 +112,12 @@ module.exports = ({ types: t, traverse }) => {
             }
           }
 
+          // https://github.com/babel/babili/issues/382
+          if (typeof res.value === "string" &&
+              res.value.indexOf("</script") > -1) {
```

Comment:
Can we use `includes` instead of `indexof` since we are already targeting node 4?